apps/web/pages/v2/settings/developer/webhooks/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { useRouter } from "next/router";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { Icon } from "@calcom/ui";
+import { Button } from "@calcom/ui/v2/core/Button";
+import Loader from "@calcom/ui/v2/core/Loader";
+import Meta from "@calcom/ui/v2/core/Meta";
+import { getLayout } from "@calcom/ui/v2/core/layouts/AdminLayout";
+
+import WebhookListItem from "@components/v2/settings/webhook/WebhookListItem";
+
+const WebhooksView = () => {
```

Comment:
Do we have an empty state for this view @joeauyeung @Jaibles ?

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/3504472/189423973-abc756a4-6abd-4c16-aebb-c9dc1c7488f6.png">

![image](https://user-images.githubusercontent.com/3504472/189424495-a43163e5-3831-4c85-b65e-e88e3dd62969.png)

Reply:
Added empty state based on routing forms:

<img width="969" alt="image" src="https://user-images.githubusercontent.com/3504472/189425385-1e72c5af-9ef2-4d2f-82fc-39a1e38d3740.png">

Reply:
sexy!